contrib/trivy/parser/v2/parser.go

```diff
@@ -34,12 +35,18 @@ func (p ParserV2) Parse(vulnJSON []byte) (result *models.ScanResult, err error)
 	return scanResult, nil
 }
 
+var dockerTagPattern = regexp.MustCompile(`:.*$`)
```

Comment:
I think tag should be more than 1 character.
```suggestion
var dockerTagPattern = regexp.MustCompile(`:.+$`)
```

Reply:
Also, if you are just judging whether or not to include `:`, I think `strings.Contains(scanResult.ServerName, ":")` is sufficient.